mlir/tools/rocmlir-lib/CMakeLists.txt

```diff
@@ -24,28 +24,28 @@ set(LIBS
 
 set(CMAKE_BUILD_RPATH ${CMAKE_BUILD_DIR}/external/llvm-project/llvm/lib)
 
-llvm_add_library(MLIRMIOpenThin
+llvm_add_library(MLIRRockThin
 PARTIAL_SOURCES_INTENDED
 
-  mlir-miopen-lib.cpp
+  rocmlir-lib.cpp
```

Comment:
I think this is the spot where we break out `rockcompiler`

That is, `rockcompiler-lib.cpp` or `librockcompiler.cpp` this directory being called `rockcompiler` or `librockcompiler` and so on

Reply:
Gonna defer on this since everything is passing.. let's do in a separate PR

Reply:
Sure

It's all fundamentally an internal change, anyhow